router/pkg/config/config.schema.json

```diff
@@ -1135,15 +1135,30 @@
           "description": "Enable error propagation. If the value is true, Subgraph errors will be propagated to the client."
         },
         "mode": {
-            "type": "string",
-            "enum": [
-                "wrapped",
-                "pass-through"
-            ],
-            "default": "wrapped",
-            "description": "The mode of error propagation. The supported modes are 'wrapped' and 'pass-through'. The 'wrapped' mode wraps the error in a custom error object to hide internals. The 'pass-through' mode returns the error as is from the Subgraph."
+          "type": "string",
+          "enum": [
+            "wrapped",
+            "pass-through"
+          ],
+          "default": "wrapped",
+          "description": "The mode of error propagation. The supported modes are 'wrapped' and 'pass-through'. The 'wrapped' mode wraps the error in a custom error object to hide internals. The 'pass-through' mode returns the error as is from the Subgraph."
+        },
+        "rewrite_paths": {
+          "type": "boolean",
+          "default": true,
+          "description": "Rewrite the paths of the Subgraph errors. If the value is true (default), the paths of the Subgraph errors will be rewritten to match the Schema of the Federated Graph."
+        },
+        "omit_locations": {
+          "type": "boolean",
+          "default": true,
+          "description": "Omit the location field of Subgraph errors. If the value is true, the location field of Subgraph errors will be omitted. This is useful because the locations of a Subgraph error is internal to the Subgraph and not relevant to the client."
+        },
+        "omit_extensions": {
+          "type": "boolean",
+          "default": false,
+          "description": "Omit the extensions field of Subgraph errors. If the value is true, the extensions field of Subgraph errors will be omitted. This is useful because the extensions of a Subgraph error is internal to the Subgraph and not relevant to the client."
```

Comment:
Could you also mention what is the default in the desc like in `rewrite_paths`? I like it.